HealthcareSystem/Backend/Service/SendingMail/MailService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Backend.Model.Users;
+using Backend.Settings;
+using MailKit.Net.Smtp;
+using MailKit.Security;
+using Microsoft.Extensions.Options;
+using MimeKit;
+
+namespace Backend.Service.SendingMail
+{
+    public class MailService : IMailService
+    {
+        private readonly MailSettings _mailSettings;
+
+        public MailService(IOptions<MailSettings> mailSettings)
+        {
+            _mailSettings = mailSettings.Value;
+        }
+        public async Task SendWelcomeEmailAsync(WelcomeRequest request)
+        {
+            string FilePath = Directory.GetCurrentDirectory() + "\\Templates\\WelcomeTemplate.html";
+            StreamReader str = new StreamReader(FilePath);
+            string MailText = str.ReadToEnd();
+            str.Close();
+            MailText = MailText.Replace("[username]", request.UserName).Replace("[email]", request.ToEmail).Replace("[jmbg]", request.Jmbg);
```

Comment:
I love that you made a template for this! I wonder if maybe you should put the path to the template into appsettings.json as well to further improve it. @vlaksi @Pufke @anciz What do you think, is that going to be necessary when we containerize this? I feel like it might be nicer anyway even if we can get it to work correctly as is.

Reply:
That is definitely something I suggest also.